src/options.js

```diff
@@ -0,0 +1,509 @@
+/* global $ */
+/*global messagesHistory, enableSync, applyTheme, userMapObj, htmlMsg*/
+var themeSelect = document.getElementById("theme");
+var clearMessageHistoryButton = document.getElementById("clearMessageHistory");
+var loginForm = document.getElementById("login");
+var logoutButton = document.getElementById("logout");
+var loginButton = document.getElementById("loginbutton");
+var noLoggedInBlock = document.getElementById("nologgedin");
+var loggedInBlock = document.getElementById("loggedin");
+var changePasswordForm = document.getElementById("changepasswordform");
+var accessToken = "";
+var userEmail = "";
+var time = "" ;
+var BASE_URL = "https://api.susi.ai";
+var messagesHistory=[];
+var userMapObj={latitude:null,longitude:null,status:null,mapids:[]};
+
+themeSelect.addEventListener("change", saveOptions);
+loginForm.addEventListener("submit", login);
+changePasswordForm.addEventListener("submit", handleChangePassword);
+logoutButton.addEventListener("click", logout);
+clearMessageHistoryButton.addEventListener("click", clearMessageHistory);
+
+document.addEventListener("DOMContentLoaded", persistSettings);
+
+function showLoggedInBlock(show){
+	if(show){
+		noLoggedInBlock.style.display="none";
+		loggedInBlock.style.display="block";
+		document.getElementById("passwordchange").value="";
+		document.getElementById("passwordnewconfirm").value = "";
+		document.getElementById("passwordnew").value = "";
+	}
+	else{
+		noLoggedInBlock.style.display="block";
+		loggedInBlock.style.display="none";
+		document.getElementById("username").value = "";
+		document.getElementById("password").value = "";
+	}
+}
+
+function login(event){
+	event.preventDefault();
+	var email=document.getElementById("username").value;
+	var password=document.getElementById("password").value;
+	if(!email){
+		alert("Email can't be blank");
+		return;
+	}
+	else if(!password){
+		alert("Password can't be blank");
+		return;
+	}
+	loginButton.innerHTML="<i class='fa fa-spinner fa-spin fa-2x fa-fw'></i>";
+	var loginEndPoint = BASE_URL+"/aaa/login.json?type=access-token&login="
+			+ encodeURIComponent(email)
+			+ "&password="
+			+ encodeURIComponent(password);
+	$.ajax({
+		url: loginEndPoint,
+		dataType: "jsonp",
+		jsonpCallback: "p",
+		jsonp: "callback",
+		crossDomain: true,
+		success: function (response) {
+			if(response.accepted){
+				accessToken = response.access_token;
+				userEmail = email;
+				browser.storage.sync.set({
```

Comment:
For chromebot accessing browser storage is done as `chrome.storage.sync.set`. Did you copied all the things? Please fix these things.